lte/gateway/c/core/oai/test/s1ap_task/test_s1ap_mme_handlers_with_injected_state.cpp

```diff
@@ -136,8 +136,8 @@ TEST_F(S1apMmeHandlersWithInjectedStatesTest, GenerateUEContextReleaseCommand) {
   ASSERT_EQ(RETURNok, generate_s1_setup_request_pdu(&pdu_s1));
 
   // State validation
-  ASSERT_TRUE(
-      is_enb_state_valid(state, assoc_id, S1AP_READY, number_attached_ue));
+  ASSERT_TRUE(is_enb_state_valid(state, assoc_id, magma::lte::oai::S1AP_READY,
```

Comment:
Please shrink the namespace reference everywhere in this while

```suggestion
  ASSERT_TRUE(is_enb_state_valid(state, assoc_id, oai::S1AP_READY,
```